examples/H2.jl

```diff
@@ -0,0 +1,34 @@
+#= Test Geometry Optimization on an aluminium supercell.
```

Comment:
@CedricTravelletti This is another prime example why too many comments can be bad :smile: ... there is nothing worse than wrong comments. If updating a comment is overlooked, it's likely not important enough to justify future maintenance cost.